BostonData/lambda_function/lambda_function.py

```diff
@@ -116,9 +116,12 @@ def get_welcome_response():
 
     session_attributes = {}
     card_title = "Welcome"
-    speech_output = "Welcome to the Boston Data skill. " \
-                    "To set your address, say " \
-                    "my address is, followed by your address."
+    speech_output = "Welcome to the Boston Public Services skill. " \
+                    "Uses include finding trash and recycling pick up days " \
+                    "for your address." \
+                    "First, set your home address," \
+                    "For example, if your address is 1 Elm street, say " \
+                    "'my address is 1 elm street"

```

Comment:
I think this is still a bit long for the welcome response. Check out the [speak and write natrually](https://developer.amazon.com/designing-for-voice/what-alexa-says/#speak-and-write-naturally) section of the Alexa design guidelines. 

We should probably have some sort of help or "what can you do" intent for the most of this info. See [providing contextual help](https://developer.amazon.com/designing-for-voice/what-alexa-says#provide-contextual-help)